Assets/Scripts/SaveSystem/SaveSystem.cs

```
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using DG.Tweening.Plugins.Core.PathCore;
+
+namespace RogueApeStudio.Crusader.SaveSystem
+{
+    public class SaveSystem : MonoBehaviour
+    {
+        [SerializeField] private SaveFileObject _saveFile;
+        public void SaveGame()
+        {
+            string saveFile = JsonUtility.ToJson(_saveFile);
+            File.WriteAllText(Application.persistentDataPath + "/SaveFile.json", saveFile);
+
+        }
+
+        public void LoadGame()
```

Is there a reason it fails silently when the file isn't found?